README.md

```diff
@@ -27,22 +27,64 @@ Multiple contact methods can be specified separated by ", " (without quotes).
 Requirements
 ------------
 
-The required Python packages are
+The required Python3 packages are
 
 * SQLAlchemy
 * sleekxmpp ≥1.0 (for the XMPP notification plugin)
-* TwitterAPI (for the Twitter notification plugin)
+* TwitterAPI twython (for the Twitter notification plugin)
 * [IRClib](https://bitbucket.org/jaraco/irc) (for the IRC notification plugin)
 
+On Debian systems install via apt with:
+
+    apt-get install python3-sqlalchemy python3-sleekxmpp python3-twython
+
+
 Setup
 -----
 
-Copy the `config.sample.py` to `config.py` and adept it to your needs. Then execute
+    git clone http://github.com/<this repositroy>/nodewatcher
+    cd nodewatcher
+    cp config.sample.py config.py
+
+and adept `config.py` to your needs.
+
+Then execute:
+
+    python3
+
 ```python
 from db import Base, engine
 Base.metadata.create_all(engine)
 ```
-to create the database structure. After that, all you need to do is call the `main.py`, for example via a cronjob in `/etc/cron.d/nodewatcher`:
+to create the database structure. 
+
+After that, all you need to do is call the `main.py`, for example via a cronjob in `/etc/cron.d/nodewatcher`:
 ```cron
-*/5 * * * * root /usr/local/src/nodewatcher/main.py
+*/5 * * * * root python3 /usr/local/src/nodewatcher/main.py
+```
+
+Database Structure
+------------------
+
+- id = internal ID for each node
+- name = hostname of the node
+- lastseen = Timestamp from nodes.json
+- contact = owner String given in the config mode of the node
+- lastcontact = Timestamp, when the owner was contacted about his offline node
+- ignore = default NULL, node owners will be sent notifications for their offline nodes:
+ - if opt_in==True and ignore=="0"
+ - if opt_in==False and (ignore=="0" or ignore==NULL)
```

Comment:
This is confusing, because it sounds as if notifications were sent by default if node owners didn't opt in.